package.json

```diff
@@ -113,18 +113,17 @@
   "dependencies": {
     "@babel/polyfill": "^7.8.7",
     "@babel/runtime": "^7.9.6",
-    "@patternfly/patternfly": "^4.10.29",
-    "@patternfly/react-core": "^3.158.1",
-    "@patternfly/react-core4": "npm:@patternfly/react-core@4.18.5",
+    "@patternfly/patternfly": "^4.10.31",
+    "@patternfly/react-core": "^4.18.5",
     "@patternfly/react-icons": "^3.15.16",
-    "@redhat-cloud-services/entitlements-client": "^1.0.63",
-    "@redhat-cloud-services/frontend-components": "1.0.28",
-    "@redhat-cloud-services/frontend-components-inventory": "1.0.33",
-    "@redhat-cloud-services/frontend-components-notifications": "^1.0.3",
-    "@redhat-cloud-services/frontend-components-remediations": "1.0.3",
-    "@redhat-cloud-services/frontend-components-utilities": "1.0.3",
+    "@redhat-cloud-services/entitlements-client": "^1.0.67",
+    "@redhat-cloud-services/frontend-components": "2.1.1",
```

Comment:
update `@redhat-cloud-services/frontend-components` to `2.1.2` and `@redhat-cloud-services/frontend-components-inventory` to `2.1.1` and we are golden!